packages/security_detection_engine/kibana/security_rule/a99f82f5-8e77-4f8b-b3ce-10c0f6afbc73.json

```
@@ -31,7 +31,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 12
+        "version": 11
```

```suggestion
        "version": 13
```
Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949